lib/approvals/writers/custom_writer.rb

```diff
@@ -0,0 +1,13 @@
+module Approvals
```

Comment:
>  I'm unsure of how the hook would end up working without a blank template for people who don't want to write their own CustomeWriter

I'm not sure what the template helps, though, since it raises exceptions. As long as you inherit from an existing Approvals thing, and you implement the two methods per the README, shouldn't that "just work" (sorry, that 'just' was ironic).

> Alternatively, what if we allowed something like verify, format: custom, "CustomWriter" 

Ah, yes. That's actually what I thought this was, because the name doesn't matter. You could inherit from the TextWriter and then pass it... but I forgot that there would probably be no way to pass it. This is a good idea.